app/web/features/communities/events/CommunityEventsList.tsx

```diff
@@ -0,0 +1,86 @@
+import { CircularProgress } from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import { CalendarIcon } from "components/Icons";
+import TextBody from "components/TextBody";
+import { useTranslation } from "i18n";
+import { COMMUNITIES } from "i18n/namespaces";
```

Comment:
This isn't actually new. I just renamed the existing `EventsList` to `CommunityEventsList` since I redesigned the old `EventsList` component, but the old one is still used on the community page.